examples/apps/ai-collab/.npmignore

```diff
@@ -0,0 +1,7 @@
+nyc
+*.log
+**/*.tsbuildinfo
+src/test
+dist/test
```

Comment:
no lib?

Reply:
Good catch, added for consistency.

Tangent: should we even have `.npmignore` on example/test apps (private: true, never packed or published; ignoring the table-document exception)?

Reply:
Good question. I think either is fine. If we have a policy checks the file exists, I wouldn't override the policy. I don't think we do though. Your call.